dcap/quoteverify/src/lib.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2022 MobileCoin Foundation
+
+#![doc = include_str!("../README.md")]
+#![deny(missing_docs, missing_debug_implementations)]
+
+use mc_sgx_dcap_quoteverify_types::Path as QvPath;
```

Comment:
Started this in the `lib.rs` module for now, I'm up for suggestions on name for a containing module